submissions/dimonlakhin/html-css-popup/index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Popup</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+    />
+    <link rel="stylesheet" href="/css/style.css" />
+    <link rel="icon" href="/img/favicon.png" />
+  </head>
+  <body>
+    <nav class="nav">
+      <ul class="nav-container">
+        <li class="nav-item">
+          <a class="nav-item__link" href="#"><span>Mail</span></a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-item__link" href="#"><span>Images</span></a>
+        </li>
+        <li class="nav-item relative">
+          <input class="check__input" type="checkbox" id="check" />
+          <label for="check">
+            <img class="nav-item__icon" src="./img/shapes.png" alt="" />
+          </label>
+          <div class="popup">
+            <ul class="container-2">
```

Comment:
Please, avoid numbers in class names, use instead [BEM](https://en.bem.info/methodology/)